apm-protocol/apm-network/src/test/java/org/apache/skywalking/api/network/trace/component/command/ServiceResetCommandTest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.api.network.trace.component.command;
+
+import org.apache.skywalking.apm.network.common.Command;
+import org.apache.skywalking.apm.network.common.KeyStringValuePair;
+import org.apache.skywalking.apm.network.trace.component.command.ServiceResetCommand;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * @author jsbxyyx
+ */
+public class ServiceResetCommandTest {
+
+    @Test
+    public void test_serialize() throws Exception {
+        ServiceResetCommand command = new ServiceResetCommand("aa");
+        Command.Builder builder = command.serialize();
+
+        Assert.assertEquals(ServiceResetCommand.NAME, builder.getCommand());
+        Assert.assertEquals("aa", builder.getArgs(0).getValue());
+    }
+
+    @Test
+    public void test_deserialize() throws Exception {
```

Comment:
I suggest splitting this test case into two cases, and the names can be more semantic such as "should_throw_when_serial_num_is_null" or "should_deserialize", which is more easy for future contributors to diagnose why they break the test and how to adjust the case to its expected goal, it's the same for your future contributions. WDYT @wu-sheng

Reply:
+1 for making test case more clear. @jsbxyyx  please follow

Reply:
OK, Your suggestion is very good.

Reply:
Hi, one reminder, using CamelCase rather than underline. We should forbit underline in method name. Contribution about this rule is welcome.
 too.

Reply:
How do you see this name? `methodName_DoesWhat_WhenTheseConditions`

Reply:
> How do you see this name? `methodName_DoesWhat_WhenTheseConditions`

Looks good to me but we prefer camel case in the project as @wu-sheng suggests, so please use the camel case style

And if the method name becomes too long or hard to describe, consider using an assert method with `message`, like `org.junit.Assert#assertEquals(java.lang.String, java.lang.Object, java.lang.Object)`, thanks :)